bioblend/galaxy/workflows/__init__.py

```diff
@@ -486,6 +492,8 @@ def invoke_workflow(
             payload["allow_tool_state_corrections"] = allow_tool_state_corrections
         if inputs_by is not None:
             payload["inputs_by"] = inputs_by
+        if not require_exact_tool_versions:
+            payload["require_exact_tool_versions"] = require_exact_tool_versions
```

Comment:
```suggestion
        payload["require_exact_tool_versions"] = require_exact_tool_versions
```

We are not doing it consistently, but I think it's better to pass boolean values in the payload unless there's a good reason not to.

Reply:
Thanks @nsoranzo, I was in two minds about that